solr/NOTICE.txt

```diff
@@ -396,6 +394,33 @@ https://github.com/rjohnsondev/java-libpst
 JMatIO is a JAVA library to read/write/manipulate with Matlab binary MAT-files.
 http://www.sourceforge.net/projects/jmatio
 
+metadata-extractor is a straightforward Java library for reading metadata 
+from image files.
+https://github.com/drewnoakes/metadata-extractor
+
+Java MP4 Parser; A Java API to read, write and create MP4 container
+https://github.com/sannies/mp4parser
+
+Jackcess; is a pure Java library for reading from and writing to MS Access 
+databases
+http://jackcess.sourceforge.net/
+
+Jackcess Encrypt; an extension library for the Jackcess project which 
+implements support for some forms of Microsoft Access and Microsoft 
+Money encryption
+http://jackcessencrypt.sourceforge.net/
+
+ROME; is a Java framework for RSS and Atom feeds
+(https://github.com/rometools/rome)
+
+VorbisJava; Ogg and Vorbis Tools for Java
+Copyright 2012 Nick Burch
+https://github.com/Gagravarr/VorbisJava
+
+SQLite JSDC Driver; is a library for accessing and creating SQLite 
```

Comment:
The last entry? SQLite

On Friday, April 22, 2016, Uwe Schindler <notifications@github.com> wrote:

> In solr/NOTICE.txt
> <https://github.com/apache/lucene-solr/pull/31#discussion_r60794497>:
>
> > +databases
> > +http://jackcess.sourceforge.net/
> > +
> > +Jackcess Encrypt; an extension library for the Jackcess project which
> > +implements support for some forms of Microsoft Access and Microsoft
> > +Money encryption
> > +http://jackcessencrypt.sourceforge.net/
> > +
> > +ROME; is a Java framework for RSS and Atom feeds
> > +(https://github.com/rometools/rome)
> > +
> > +VorbisJava; Ogg and Vorbis Tools for Java
> > +Copyright 2012 Nick Burch
> > +https://github.com/Gagravarr/VorbisJava
> > +
> > +SQLite JSDC Driver; is a library for accessing and creating SQLite
>
> This is a typo, I think.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/lucene-solr/pull/31/files/77428cd3e1a38a8a3bc3920d3a1e80116440e3c6#r60794497>
>


-- 
*Lewis*